digicampipe/io/containers.py

```diff
@@ -156,7 +191,16 @@ class R0CameraContainer(Container):
     local_camera_clock = Field(np.int64, "camera timestamp")
     gps_time = Field(np.int64, "gps timestamp")
     white_rabbit_time = Field(float, "precise white rabbit based timestamp")
-    camera_event_type = Field(int, "camera event type")
+    _camera_event_type = Field(CameraEventType, "camera event type")
+
+    @property
```

Comment:
I don't understand why we have 2 "camera_event_type"s one is "private" the other is "public".
If only 1 is required I would keep the "public" one

Reply:
what do you mean by two? the red line was removed ... the green line now exists... the public was converted into a private and needs to go through a setter and getter now, so the type is correctly converted.

Reply:
> Sorry to ask again 
did you ask that already before? ... see it the first time... maybe... I missed it before sorry.

Good question... maybe ... In fact, maybe you want to have a look at this from the CamerasToACTL repo:

https://forge.in2p3.fr/projects/ctaactl/repository/entry/CamerasToACTL/trunk/DataModel/L0.proto#L207

The File `L0.proto` contains the protobuf data model, which describes how the for example the `CameraEvent` looks like (line 329 of L0.proto). The link above goes to line 207 of the file, where an enumeration is defined for exactly what we are discussing here.

We observe two things immediately:

 * SST1M does not use this.
 * This enum does not define a bitmap encoding, where and event can be ALG0 **and** CONT at the same time, but here the type is either PHYSICAL **or** PEDESTAL.

If we look deeper into it, we see the enum from line 207 named `EventType` is used for the field named `eventType` defined in line 333. Which field are we using then?? Ah of course .. we are using the field `event_type` which is not an enum, but a simple integer...why? Well that's obvious, because the `EventType` enum was not suitable for our use case...

Anyway somewhere down the line people took decisions.... and now we (at pretty much the end of the line) have to live with it... but asking us to live with it in clearly readable code, which is easy to understand? Well I guess that is not so easy, since the whole problem is not easy to understand in the first place.

Reply:
if we don't use `event_type` shouldn't we just remove it from the container?